docker/sklearn/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM demisto/python3:3.7.5.5420
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git g++ python3-dev\
+        openssh curl ca-certificates openssl less htop \
+		make  rsync \
+        libpng-dev freetype-dev libexecinfo-dev libgomp lapack-dev \
+		libgcc libquadmath musl  \
+		libgfortran \
+        gcc \
+        python py-pip  \
+        gfortran \
+        musl-dev \
+  && apk add --no-cache libstdc++  \
+  && apk add --no-cache openblas-dev  \
```

Comment:
I think you can move this up to be part of the install which is for `.build-dependencies`

Reply:
I can't. If I remove those 2 packages the code won't work. 
libstdc++ is necessary for pandas 
openblas-dev for skikit learn 
@glicht

Reply:
I am not too familiar with this. But `openblas-dev` is a dev pacakge. For runtime you should probably be using `openblas`. I would try adding `openblas-dev` to the build-deependencies and installing separately `openblas`.